src/AUTOEXP.DAT

```diff
@@ -1,2066 +0,0 @@
-;---------------------------------------------------------------
-; This comment block added by the CLR dev team.
```

Comment:
End of an era..do these not work anymore?

Reply:
I don’t know if this doesn’t work anymore, but Visual Studio has a new way of doing debug visualizers with natvis files so that we don’t have to change our VS installation. And WinDbg also supports them. https://docs.microsoft.com/en-us/visualstudio/debugger/create-custom-views-of-native-objects?view=vs-2019

Reply:
@danmosemsft These files are still supported, but `.natvis` files are now preferred. The issue with the `AUTOEXP.DAT` and native visualizers in general is that they are particularly sensitive to type layouts. As soon as one aspect of the type visualizer layout changes other aspects of the type visualizer can fail and `AUTOEXP.DAT` have a particularly poor diagnostic experience.